src/app/features/event/event.state.service.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpClient } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { ENDPOINTS } from "@shared/constants";
+
+import { BehaviorSubject } from "rxjs";
+import { Maybe } from "@shared/utility-types";
+import { EventModule } from "./event.module";
+import { EventCardDTO, EventDetailsDTO } from ".";
+
+export type EventState = {
+  eventDetails: Maybe<EventDetailsDTO>;
+  eventsList: EventCardDTO[];
+};
+
+const eventStateDefault = {
+  eventDetails: null,
+  eventsList: [],
+};
+
+@Injectable({
+  providedIn: EventModule,
+})
+export class EventStateService {
```

Comment:
celowa decyzja by nie używać `ngrx` ?

Reply:
Tak, lokalnie będziemy używać serwisów, globalnie stora. Struktura tego serwisu pewnie jeszcze się zmieni, chodziło o to aby mieć serwis do pobierania danych o eventach.